scripts/onboarding/aks/onboarding-msi-bicep-syslog/existingClusterOnboarding.bicep

```diff
@@ -39,6 +39,9 @@ param streams array
 @description('The flag for enable containerlogv2 schema')
 param enableContainerLogV2 bool
 
+@description('Enable Retina Network Flow Logs in omsagent addon profile')
```

Comment:
@wanlonghenry , did you verify both network flow logs and high log scale scenarios?

Reply:
yes tested both 
for networkflow logs, tested with parameter setting networkflow logs true
for highlogs scale, tested with setting configmap and stream